.golangci.yml

```diff
@@ -0,0 +1,22 @@
+# golangci-lint configuration used for CI
+run:
+  tests: true
+  timeout: 10m
+  skip-files:
+    - ".*\\.pb\\.go"
```

Comment:
pb not required I guess?

Do we need to add .csv for antrea registry?

Reply:
Got it. Will remove this line(for skipping `.pb.go` files). it will skip code that is not in Golang so we don't need to add `.csv` here. Thanks!